### PR TITLE
docs: added contributing and developers documentation - #50

### LINKED ISSUE #50: Adding `CONTRIBUTNG.md` and `DEVELOPERS.md` files
# Discussion 🗣
Adding `CONTRIBUTNG.md` and `DEVELOPERS.md` files in the repo similar to other Accord Project's repositories
These files will point out to techdocs [CONTRIBUTNG.md](https://github.com/accordproject/techdocs/blob/master/CONTRIBUTING.md) and [DEVELOPERS.md](https://github.com/accordproject/techdocs/blob/master/DEVELOPERS.md) respectively

Also, we can add `Cicero-word-add-in` specific information in them similar to what it is done in Cicero's [CONTRIBUTNG.md](https://github.com/accordproject/cicero/blob/master/CONTRIBUTING.md#cicero-specific-information)

I would like to create a PR for this if needed

### PR DESCRIPTION
**Pull request by Aniruddha-Shriwant**
Signed-off-by: Aniruddha Shriwant <aniruddhashriwant@gmail.com>

# Closes #50 <CORRESPONDING ISSUE NUMBER>

### Changes

- Added `CONTRIBUTING.md` & `DEVELOPERS.md`
Check [this branch](https://github.com/Aniruddha-Shriwant/cicero-word-add-in/tree/docs-contributing) for preview

### Author Checklist
- [x] Ensure you provide a [DCO sign-off](https://github.com/probot/dco#how-it-works) for your commits using the `--signoff` option of git commit.
- [ ] Vital features and changes captured in unit and/or integration tests
- [x] Commits messages follow [AP format](https://github.com/accordproject/techdocs/blob/master/DEVELOPERS.md#commit-message-format)
- [x] Extend the documentation, if necessary
- [x] Merging to `master` from `fork:branchname`
- [ ] Manual accessibility test performed
    - [ ] Keyboard-only access, including forms
    - [ ] Contrast at least WCAG Level A
    - [ ] Appropriate labels, alt text, and instructions

### CODE REVIEW

File: DEVELOPERS.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+# Cicero Word Add-in Development Guide
+
+## ❗ Accord Project Development Guide ❗
+We'd love for you to help develop improvements to Cicero Word Add-in! Please refer to the [Accord Project Development guidelines][apdev] we'd like you to follow.
+
+## Cicero Word Add-in Specific Information
```
Comment (algomaster99) [inline DEVELOPERS.md:3]: All the information below is already in the README so no point repeating it. I would suggest to keep this but delete the instructions in the README but provide a link there to this markdown file.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Accord Project Cicero Word Add-in Contribution Guide
```
Comment (algomaster99) [inline CONTRIBUTING.md:1]: Provide a link to this guide in the README as well.

**Review by algomaster99 — CHANGES_REQUESTED** [review]
Looks nice! I dropped my comments.

File: DEVELOPERS.md (hunk 1 above)
Comment (Aniruddha-Shriwant) [reply DEVELOPERS.md:3]: > so no point repeating it.

Yeah 😄 

>  I would suggest to keep this but delete the instructions in the README but provide a link there to this markdown file.

Sure I will do these changes soon.

File: CONTRIBUTING.md (hunk 2 above)
Comment (Aniruddha-Shriwant) [reply CONTRIBUTING.md:1]: Okay 👍🏻

Comment (Aniruddha-Shriwant) [reply CONTRIBUTING.md:1]: @algomaster99, as suggested there is already a hyperlink present for these files in the readme in [this](https://github.com/accordproject/cicero-word-add-in#contributing) section so I don't think there is a need to provide this link at top of readme again. Still, I'm willing to know what are your views on this

Comment (algomaster99) [reply CONTRIBUTING.md:1]: Oh okay. We won't need to add link in the README then if it already exists.

Comment (Aniruddha-Shriwant) [reply CONTRIBUTING.md:1]: Yeah, So I will update the readme by deleting the [`For Developers`](https://github.com/accordproject/cicero-word-add-in#for-developers) section

Comment (Aniruddha-Shriwant) [reply CONTRIBUTING.md:1]: @algomaster99, I had deleted the [ `For Developers` ](https://github.com/accordproject/cicero-word-add-in#for-developers) section now the readme looks like [this](https://github.com/Aniruddha-Shriwant/cicero-word-add-in/tree/docs-contributing#readme). We may update the readme again by including some more information related to the add-in, similar to `accordproject/template-studio` Readme's [`For Contributors`](https://github.com/accordproject/template-studio#for-contributors) section. 
For example:



# For contributors

The Cicero Word Add-in is a work in progress and we are very interested to hear from you. Let us know what you like or don't like about the Add-in, report bugs, or suggest new features by opening a GitHub issue, and Consider contributing to the project as we are very contributory-friendly! 🙂

---

**Review by algomaster99 — CHANGES_REQUESTED** [review]
In my opinion, the README after the changes ends very abruptly. I think it would be better to link `DEVELOPERS.md` and `CONTRIBUTING.md`. I agree that these documents have already been linked in the second section which talks about AP in general. But it seems a little out of place because section 1 (Cicero Word Add-in) should talk about only Cicero Word Add-in and section 2 (Accord Project logo) should talk about AP broadly.

What you can do to rectify this:
1. Add links to the `DEVELOPERS.md` and `CONTRIBUTING.md` in section 1.
2. Replace links in section 2 corresponding to the markdown files here - https://github.com/accordproject/techdocs.

**Comment by Aniruddha-Shriwant** [comment]
> In my opinion, the README after the changes ends very abruptly.

Agreed! I will make the requested changes

**Comment by Aniruddha-Shriwant** [comment]
@algomaster99 I had updated the `README.md` according to the requested changes.
Have a look at the new README here:
https://github.com/Aniruddha-Shriwant/cicero-word-add-in/tree/docs-contributing#readme
any suggestions are welcome :)

File: README.md [hunk 3]
```diff
@@ -130,8 +65,11 @@ Copyright 2018-2019 Clause, Inc. All trademarks are the property of their respec
 [apdoc]: https://docs.accordproject.org/
 [apslack]: https://accord-project-slack-signup.herokuapp.com
 
-[contributing]: https://github.com/accordproject/cicero-word-add-in/blob/master/CONTRIBUTING.md
-[developers]: https://github.com/accordproject/cicero-word-add-in/blob/master/DEVELOPERS.md
+[contributing]: https://github.com/accordproject/techdocs/blob/master/CONTRIBUTING.md
+[developers]: https://github.com/accordproject/techdocs/blob/master/DEVELOPERS.md
+
+[addindevelopers]: https://github.com/accordproject/cicero-word-add-in/blob/master/DEVELOPERS.md
```
Comment (algomaster99) [inline README.md]: Refrain from using absolute URLs for files contained in this repository. You can simple use 'DEVELOPERS.md'.

File: DEVELOPERS.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
+# Cicero Word Add-in Development Guide
+
+## ❗ Accord Project Development Guide ❗
```
Comment (algomaster99) [inline DEVELOPERS.md]: Sorry, I did not point this out before. But it is better to keep this below `Cicero Word Add-in Specific Information` as we want to highlight cicero-word-add-in in this README and not general AP development.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+
+## ❗ Accord Project Contribution Guide ❗
```
Comment (algomaster99) [inline CONTRIBUTING.md]: Switch the headings here too.

**Review by algomaster99 — CHANGES_REQUESTED** [review]
Few more changes and we would be good to go.

File: DEVELOPERS.md (hunk 4 above)
Comment (Aniruddha-Shriwant) [reply DEVELOPERS.md]: Yes agreed, I had updated the files

**Review by algomaster99 — APPROVED** [review]
Going for merge!

**Pull request merged**